practice-app/backend.py

```diff
@@ -68,30 +68,24 @@ def api_authorstats():
     return jsonify(json)  
 
 
-@app.route('/profile',methods=['POST'])
-def profile():
-    if request.method=='POST':
-        authorJson=scholar_util.search_authors_by_name(request.form["name"])
-        print(authorSearchResult)
+@app.route('/profile/<name>', methods=['GET'])
+def profile(name):
 
-        #TODO:Get these info and add them to profile.html page
-        scholar_util.getNameOutOfAuthorJson(authorJson)
-        scholar_util.getAuthorsColloborators(authorJson)
-        scholar_util.getAuthorsCitationIndexes(authorJson)
-        scholar_util.getAuthorPhoto(authorJson)
-        scholar_util.getAuthorsRecentPublications(authorJson)
+    url = request.url_root+'api/profiledata?name=' + name
```

Comment:
Since there is already a function directly returning the data, I believe it's redundant so use another request to get it. Directly calling that function would be better.

Reply:
I think api call should be used in order to demonstrate that our api is usable. If we call the function directly there is no point to have an api end point for such functionality.

Reply:
Yes, that's debatable. I think we need others' opinions on subject.

Reply:
We agreed on both of those approaches are applicable, since both of them are able to demonstrate API functionalities.